frontend/src/scenes/persons/RelatedFeatureFlags.tsx

```diff
@@ -158,11 +158,13 @@ export function RelatedFeatureFlags({ distinctId, groups }: Props): JSX.Element
                         <b>Match evaluation</b>
                     </span>
                     <LemonSelect
-                        options={[
-                            { label: 'All', value: 'all' },
-                            { label: 'Matched', value: FeatureFlagMatchReason.ConditionMatch },
-                            { label: 'Not matched', value: 'not matched' },
-                        ]}
+                        options={
+                            [
+                                { label: 'All', value: 'all' },
+                                { label: 'Matched', value: FeatureFlagMatchReason.ConditionMatch },
+                                { label: 'Not matched', value: 'not matched' },
+                            ] as { label: string; value: string }[]
```

Comment:
I think `LemonSelect` has types already specific for this instead of needing `{ label: string; value: string }`

Reply:
It does but I think Typescript gets confused when it only sees a single string value (the rest being enums), and basically sets the type of the options passed in to be `{ label: string; value: "all" }[]`. So we just have to tell it that it's more generic and then LemonSelect will do it's thing from there

Reply:
Going to catch up with @Twixes because there is some more difficult cases with LemonTable